### PR TITLE
virsh_sendkey: adopt sysrq tests to use /var/log/syslog for ubuntu

### PR DESCRIPTION
**Pull request by balamuruhans**
check for VM distribution and use /var/log/syslog for ubuntu instead
of /var/log/messages for the test to proceed seamless

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_sendkey.py [hunk 1]
```diff
@@ -58,14 +58,17 @@ def send_line(send_str):
     session = vm.wait_for_login()
 
     if sysrq_test:
+        log_file = "/var/log/messages"
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_sendkey.py]: maybe set it as global since it is used by multiple places

**Comment by balamuruhans** [comment]
@chunfuwen thanks for the review, I have made the changes suggested,

**Job log:**
```
JOB ID     : 8d09c5137a7db89ef8c4cc1dea0fc767d324a667
JOB LOG    : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2018-03-28T05.00-8d09c51/job.log
 (01/15) virsh_sendkey.guest_import.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-qemu.unattended_install.import.import.default_install.aio_native:  PASS (65.56 s)
 (02/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.without_codeset:  PASS (62.12 s)
 (03/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.linux_keycode:  PASS (14.31 s)
 (04/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.at_set1_keycode:  PASS (14.21 s)
 (05/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.at_set2_keycode:  PASS (14.35 s)
 (06/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.at_set3_keycode:  PASS (14.33 s)
 (07/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.xt_keycode:  PASS (14.21 s)
 (08/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.xt_kbd_keycode:  PASS (14.18 s)
 (09/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.usb_keycode:  PASS (14.23 s)
 (10/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.rfb_keycode:  PASS (14.30 s)
 (11/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.default_name:  PASS (14.33 s)
 (12/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.help:  PASS (16.99 s)
 (13/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.show_memory_usage:  PASS (16.71 s)
 (14/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.show_task_status:  FAIL (102.71 s)
 (15/15) virsh_sendkey.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.reboot_guest:  FAIL (220.03 s) 
RESULTS    : PASS 13 | ERROR 0 | FAIL 2 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 618.03 s
JOB HTML   : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2018-03-28T05.00-8d09c51/results.html
```

**Comment by balamuruhans** [comment]
Fixed the failures observed above and included the commit in this PR,
```
JOB ID     : 6613b6a0312dad063517aa95afd7b53e6e042163
JOB LOG    : /var/lib/libvirt/images/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2018-03-29T10.59-6613b6a/job.log
 (01/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.without_codeset:  PASS (62.12 s)
 (02/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.linux_keycode:  PASS (14.67 s)
 (03/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.at_set1_keycode:  PASS (14.77 s)
 (04/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.at_set2_keycode:  PASS (14.65 s)
 (05/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.at_set3_keycode:  PASS (14.66 s)
 (06/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.xt_keycode:  PASS (14.66 s)
 (07/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.xt_kbd_keycode:  PASS (14.71 s)
 (08/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.usb_keycode:  PASS (14.80 s)
 (09/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.rfb_keycode:  PASS (14.67 s)
 (10/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.params_test.non_acl.default_name:  PASS (14.76 s)
 (11/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.help:  PASS (17.18 s)
 (12/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.show_memory_usage:  PASS (17.14 s)
 (13/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.show_task_status:  PASS (17.39 s)
 (14/14) guest_lifecycle.sendkey.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.virsh.sendkey.sysrq.non_acl.reboot_guest:  PASS (88.83 s)
RESULTS    : PASS 14 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 341.89 s
JOB HTML   : /var/lib/libvirt/images/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2018-03-29T10.59-6613b6a/results.html

```

**Comment by balamuruhans** [comment]
Thanks @chunfuwen for the review.

**Pull request merged**